### ISSUE TITLE
Additional notes for Mensa script

### ISSUE THREAD

**Issue by dirkonet** [fsr/matthias#9]
Please add shortened versions of the additional information (Schwein, Rind, Vegetarisch, Vegan, Alkohol, Knoblauch,… – e.g. S/R/V/V+/A/K) at the end after the price. Maybe, while doing that, also change the price display to double cent digits.

**Comment by kiliankoe** [comment]
sounds like a good idea, will do later

**Comment by kiliankoe** [comment]
Both done in 6b88bfa58ba58b0525c528668d07c97bcce7e520

**Comment by dirkonet** [comment]
Another idea: Emojis. :pig: :cow: :wine_glass: and additional carrot and leaf emoticons.

**Comment by heisig** [comment]
Good Idea! I would suggest 🌿 for vegetarian meals hence carrot is yet to be added in Unicode (expected mid 2016). No idea for the garlic one. There is not even a vampire..

**Comment by dirkonet** [comment]
:green_apple: or :apple: for vegetarian, 🌿 for vegan? Bats are also not yet added :( :nose: for garlic?

**Comment by heisig** [comment]
Oh yeah, I forgot about vegan. After checking the representation in our slack i would use

:pig: for pork
:cow: for beef
:wine_glass: for alcohol
🍅 for vegetarian
🌿 for vegan

The nose for garlic isn't that great in my opinion. It's ugly and we have to see it in most of the meals.. we could just drop it form the list. Who has a problem with garlic?

**Comment by kiliankoe** [comment]
Slack supports custom emoji, so we could just add our own icon that looks similar.

**Comment by dirkonet** [comment]
Would it be possible to match words in the meal name and display icons accordingly? Hühnchen, Hähnchen, Pute,… could be :chicken: or :poultry_leg:, Hirsch and Wild could be :bambi: or similar.

**Comment by JustusAdam** [comment]
I don't think that is a good idea. It would severely impede the readability in my opinion.

**Comment by dirkonet** [comment]
Why would a single additional icon behind the price impede the readability? It would just expand what we already do for pork and beef to other types of meat and thereby improve readability. While we are at it, :salmon: and :fish: maybe?

**Comment by kiliankoe** [comment]
Dirk isn't talking about replacing the words in the text, but about adding extra icons at the end. Imo that would look just fine.
